relayer/docker.go

```diff
@@ -445,6 +446,20 @@ func (r *DockerRelayer) StopRelayer(ctx context.Context, rep ibc.RelayerExecRepo
 	return nil
 }
 
+func (r *DockerRelayer) PauseRelayer(ctx context.Context) error {
+	if r.containerLifecycle == nil {
+		return nil
+	}
+	return r.client.ContainerPause(ctx, r.containerLifecycle.ContainerID())
+}
+
+func (r *DockerRelayer) ResumeRelayer(ctx context.Context) error {
+	if r.containerLifecycle == nil {
+		return nil
+	}
+	return r.client.ContainerUnpause(ctx, r.containerLifecycle.ContainerID())
+}
```

Comment:
`r.containerLifecycle == nil` seems suspicious. Is nil a valid state for this struct member? Would we want to panic or return an error to fail fast?

Reply:
yes, stop do the same, `r.containerLifecycle` is set to nil for stopped container.